src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -32,6 +32,8 @@ def create_rss_notifications(event)
         event_payload: event.payload,
         subscription_receiver_role: subscription.receiver_role
       }
+
+      next if subscription.subscriber && subscription.subscriber.away?
```

Comment:
Same as last commit: please explain the reason of this commit in the body of the commit message. Or merge this commit into the previous ones, as you prefer.

Reply:
The PR is pretty descriptive, the code itself is clear.. could you please mark it as nitpick?

Reply:
Moved to nitpick.